tikv/tikv
# remote empty log

### What problem does this PR solve?

Issue Number: close #7327  

Problem Summary: extra empty line when Tikv startup

### What is changed and how it works?

Filter empty line before print

### Related changes

None

### Check List 

Tests 

- Manual test
  start `cmd/bin/tikv-server.rs` to check output log
![Screenshot from 2020-04-01 22-44-21](https://user-images.githubusercontent.com/12531298/78205545-62226c80-746a-11ea-8845-89d4ac4f6ff9.png)


Side effects

None

### Release note
